src/test/resources/po_line.sample

```diff
@@ -10,6 +10,8 @@
   "claims": [
     "f7d7d4e4-255e-46dc-9f08-b953fb1e872d"
   ],
+  "checkin_items": false,
```

Comment:
again, https://github.com/folio-org/mod-orders-storage/pull/90 renames these files and refactors them to use camelCase.  You'll have to adjust once those changes are merged

Reply:
I don't see `"checkinItems": false` in the (renamed) `po-line.sample`

Reply:
Missed. Thanks